src/sdk/publicDecrypt.ts

```diff
@@ -0,0 +1,68 @@
+import { bytesToBigInt, fromHexString } from '../utils';
+import {
+  u8vec_to_cryptobox_pk,
+  new_client,
+  process_reencryption_resp_from_js,
+  u8vec_to_cryptobox_sk,
+} from 'node-tkms';
+import { ethers, getAddress } from 'ethers';
+
+const aclABI = [
+  'function persistAllowed(uint256 handle, address account) view returns (bool)',
+];
+
+export const publicDecryptRequest =
+  (
+    kmsSignatures: string[],
+    chainId: number,
+    kmsContractAddress: string,
+    aclContractAddress: string,
+    relayerUrl: string,
+    provider: ethers.JsonRpcProvider | ethers.BrowserProvider,
+  ) =>
+  async (handle: bigint) => {
+    const payloadForRequest = {
+      ciphertext_handle: handle.toString(16).padStart(64, '0'),
+    };
+    const options = {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(payloadForRequest),
+    };
+
+    let response;
+    let json;
+    try {
+      response = await fetch(`${relayerUrl}publicDecrypt`, options);
```

Comment:
explicit / looks better and is more reliable in case of config forgetting it

Reply:
Sure, since the function is not implemented yet I use this one but we need to discuss with relayer to see what will be the url :)